ImgShareData/ImgurEndpoints.cs

```diff
@@ -1,34 +1,34 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace ImgShare.APISource.Data
-{
-    static public class ImgurEndpoints
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImgShare.APISource.Data
+{
+    static public class ImgurEndpoints
     {
-        #region API Base
-        static internal String APIBase
-        {
-            get
-            {
-                switch (_useType)
-                {
-                    case ImgurEndpointUseType.free:
-                        return _freeEndpoint;
-                    case ImgurEndpointUseType.paid:
-                        return _paidEndpoint;
-                    default:
-                        return _freeEndpoint;
-                }
-            }
-        }
-
-        static private String _freeEndpoint = "https://api.imgur.com/3/{0}";
+        #region API Base
+        static internal String APIBase

```

Comment:
nit: a bit of documentation about this might be useful. i'm thinking like `returns the api base endpoint depending on the type of usage you have (premium vs free)`